src/modes/summary/Summary.svelte

```diff
@@ -69,18 +68,15 @@
   </RegionDatePicker>
   <div class="uk-container content-grid">
     <div class="grid-3-11">
-      <FancyHeader invert>{region.displayName}</FancyHeader>
```

Comment:
Should we leave the region name in there for consistency, perhaps followed by an `<hr />`?  This will appear in the "Location" view of this page, after all.